README.md

```diff
@@ -121,39 +121,20 @@ The following tables lists the configurable parameters of the Orchestrate chart
-| `keyManager.resources.limits`            | The resources limits for Orchestrate keyManager containers                                                             | `{}`        |
-| `keyManager.resources.requests`          | The requested resources for Orchestrate keyManager containers                                                          | `{}`        |
-| `keyManager.nodeSelector`                | Node labels for pod assignment                                                                                         | `{}`        |
-| `keyManager.tolerations`                 | Tolerations for pod assignment                                                                                         | `[]`        |
-| `keyManager.affinity`                    | Affinity for pod assignment                                                                                            | `{}`        |
-| `keyManager.environment`                 | Environment variables passed to Orchestrate keyManager containers                                                      | `{}`        |
-| `keyManager.environmentSecrets`          | Environment variables (as Kubernetes secrets) passed to Orchestrate Key Manager containers                             | `{}`        |
-| `keyManager.existingSecret`              | If specified, extra environment variables will be added from externally created secret                                 | `nil`       |
-| `keyManager.migrate`                     | Run import keys from Vault formated with Orchestrate v2.5.X to Orchestrate Vault plugin (used in v21.1.X)              | `false`     |
-| `keyManager.vaultAgent.enabled`          | Run Vault Agent to retrieve the Key Manager client token from Hashicorp Vault                                          | `false`     |
-| `keyManager.vaultAgent.role`             | Role eligible to retrieve a token                                                                                      | `client`    |
-| `keyManager.vaultAgent.image.repository` | Hashicorp Vault image                                                                                                  | `vault`     |
-| `keyManager.vaultAgent.image.tag`        | Hashicorp Vault tag                                                                                                    | `1.6.2`     |
-| `keyManager.vaultAgent.config.wrapTTL`   | Response-wrapped TTL, see https://www.vaultproject.io/docs/agent/autoauth                                              | ``          |
+| `qkm.enabled`                     | Deploy Quorum Key Manager                                                                                                     | `true`      |
+| `qkm.fullname`                     | Name Manager                                                                                                     | `quorumkeymanager`      |
```

Comment:
you are missing the `KEY_MANAGER_STORE_NAME` to define which store of QKM is used by Orchestrate

Reply:
YOu are missing the authentication method to authenticate Orchestrate in QKM